cpp/unreal_plugins/SpEngine/Source/SpEngine/GameWorldService.h

```diff
@@ -41,6 +64,120 @@ class GameWorldService {
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "find_property_by_name_from_object", [this](const uint64_t& ptr, const std::string& name) -> GameWorldServicePropertyDesc {
+            Unreal::PropertyDesc property_desc = Unreal::findPropertyByName(reinterpretAs<UObject*>(ptr), name);
+            GameWorldServicePropertyDesc game_world_property_desc;
+            game_world_property_desc.property_ = reinterpretAs<uint64_t>(property_desc.property_);
+            game_world_property_desc.value_ptr_ = reinterpretAs<uint64_t>(property_desc.value_ptr_);
+            return game_world_property_desc;
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "find_property_by_name_from_struct", [this](const uint64_t& value_ptr, const uint64_t& struct_ptr, const std::string& name) -> GameWorldServicePropertyDesc {
+            Unreal::PropertyDesc property_desc = Unreal::findPropertyByName(reinterpretAs<void*>(value_ptr), reinterpretAs<UStruct*>(struct_ptr), name);
+            GameWorldServicePropertyDesc game_world_property_desc;
+            game_world_property_desc.property_ = reinterpretAs<uint64_t>(property_desc.property_);
+            game_world_property_desc.value_ptr_ = reinterpretAs<uint64_t>(property_desc.value_ptr_);
+            return game_world_property_desc;
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_property_value_as_string", [this](const GameWorldServicePropertyDesc& service_property_desc) -> std::string {
+            Unreal::PropertyDesc property_desc;
+            property_desc.property_ = reinterpretAs<FProperty*>(service_property_desc.property_);
+            property_desc.value_ptr_ = reinterpretAs<void*>(service_property_desc.value_ptr_);
+            return Unreal::getPropertyValueAsString(property_desc);
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "set_property_value_as_string", [this](const GameWorldServicePropertyDesc& service_property_desc, const std::string& string) -> void {
+            Unreal::PropertyDesc property_desc;
+            property_desc.property_ = reinterpretAs<FProperty*>(service_property_desc.property_);
+            property_desc.value_ptr_ = reinterpretAs<void*>(service_property_desc.value_ptr_);
+            Unreal::setPropertyValueFromString(property_desc, string);
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "find_function_by_name", [this](const uint64_t& ptr, const std::string& name, const EIncludeSuperFlag::Type& include_super_flag) -> uint64_t {
+            return reinterpretAs<uint64_t>(Unreal::findFunctionByName(reinterpretAs<UClass*>(ptr), name, include_super_flag));
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "call_function_without_args", [this](const uint64_t& object_ptr, const uint64_t& function_ptr) -> TMapStringToString {
```

Comment:
Don't need this entry point. You can handle the no-argument case at the Python level.